example/android/app/build.gradle

```diff
@@ -39,7 +39,7 @@ android {
     defaultConfig {
         // TODO: Specify your own unique Application ID (https://developer.android.com/studio/build/application-id.html).
         applicationId "np.com.naxa.graphhooper_route_example"
-        minSdkVersion 16
+        minSdkVersion 20
```

Comment:
Why did you bump this?

Reply:
I think while I was trying to include Write Permissions in the manifest, there was error regarding the minSdk. Will review it.